src/backend/InvenTree/plugin/builtin/labels/label_sheet.py

```diff
@@ -149,7 +151,7 @@ def print_labels(
         # Render to a single HTML document
         html_data = self.wrap_pages(pages, **document_data)
 
-        if self.in_debug_mode():
```

Comment:
I think this would read cleaner if you flipped the cases rather than negating the logic check